api/v1alpha1/ingressnodefirewallconfig_types.go

```diff
@@ -21,27 +21,26 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
-// IngressNodeFirewallConfigSpec defines the desired state of IngressNodeFirewallConfig
+// IngressNodeFirewallConfigSpec defines the desired state of IngressNodeFirewallConfig.
 type IngressNodeFirewallConfigSpec struct {
-	// Nodes where the daemonset will be running on.
+	// nodeSelector is used to select which Nodes the ingress node firewall DaemonSet will be run on.
+	// + This field is using map[string]string instead of metav1.LabelSelector because it matches the daemonset NodeSelector template. Thus one cannot benefit from additional match criteria that metav1.LabelSelector provides.
 	// +optional
 	NodeSelector map[string]string `json:"nodeSelector,omitempty"`
-	// tolerations is a list of tolerations applied to ingress node firewall controller
-	// deployments.
+	// A list of tolerations applied to ingress node firewall controller.
```

Comment:
remove the full stop at the end

Reply:
I am sure there will be more comments from Joel so I can defer this change to the follow up PR(s) ok ?

Reply:
yep

Reply:
I don't know what the OpenShift API standard is, but all comments in Golang should be complete phrases and have a full stop at the end of a phrase. 
https://github.com/golang/go/wiki/CodeReviewComments#comment-sentences

Reply:
It's just that right now it reads ... ;-)  So we'll get something like: `A list of tolerations applied to ingress node firewall controller. daemonsets.`  in the explain
// A list of tolerations applied to ingress node firewall controller.
// daemonsets.

Reply:
feel free to edit it in any language u like @andreaskaris once its merge :)